test/views/pages/ip2016/PsoDetailsViewSpec.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.pages.ip2016
+
+import forms.PSODetailsForm
+import org.jsoup.Jsoup
+import play.api.i18n.Messages.Implicits._
+import testHelpers.ViewSpecHelpers.CommonViewSpecHelper
+import testHelpers.ViewSpecHelpers.IP16.PsoDetailsViewMessages
+import views.html.pages.ip2016.{psoDetails => views}
+
+class PsoDetailsViewSpec extends CommonViewSpecHelper with PsoDetailsViewMessages {
+
+  "the PsoDetailsView" should{
+    val pensionsForm = PSODetailsForm.psoDetailsForm.bind(Map(
+      "psoDay" -> "1",
+      "psoMonth" -> "2",
+      "psoYes" -> "2017",
+      "psoAmt" -> "12345"))
+
+    lazy val view = views(pensionsForm)
+    lazy val doc = Jsoup.parse(view.body)
+
+    val errorForm =  PSODetailsForm.psoDetailsForm.bind(Map.empty[String, String])
+    lazy val errorView = views(errorForm)
+    lazy val errorDoc = Jsoup.parse(errorView.body)
+
+    lazy val form = doc.select("form")
+
+    "have the correct title" in{
+      doc.title() shouldBe plaPsoDetailsTitle
+    }
+
+    "have the correct and properly formatted header"in{
+      doc.select("h1").text shouldBe plaPsoDetailsTitle
+      doc.select("h1").hasClass("heading-large") shouldBe true
+    }
+
+    "have the right subheadings" in{
```

Comment:
...I know these are tagged as subheadings but they aren't actually. Please rewrite the wording to match the actual labels/legends they represent.